src/common.js

```diff
@@ -367,11 +369,13 @@ function wrap(cmd, fn, options) {
           if (e.msg === 'earlyExit') {
             retValue = e.retValue;
           } else {
+            /* istanbul ignore next */
```

Comment:
Yeah, either way. I just usually stick with `ignore next`. Do you think I should change it?